timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+click the `VPC` tab within the `Operations` view. From this view, you have a few options
+depending on the state of your service.
+
+<highlight type="warning">
+DNS propagation can take time. After migrating your Timescale Forge service,
+please allow a few minutes for DNS propagation. If you receive DNS errors indicating
+that the DNS name could not be resolved, this indicates that more time is needed for
+DNS propagation.
+</highlight>
+
+### Migrate from public network to VPC
+When your service is attached to the public network, you have the option to select a VPC
+to migrate your service into.
+
+<img class="main-content__illustration" src="migrate-public-to-vpc.png" alt="Migrate from public network to VPC"/>
+
+Once you have selected the VPC to migrate your service into, click `Attach VPC`. When prompted
+for confirmation, type `MIGRATE` as instructed, then click `Migrate`.
+
+<img class="main-content__illustration" src="migrate-public-to-vpc-confirm.png" alt="Confirm migration into VPC"/>
+
+Your service will now go through a series of asynchronous operations ultimately
+resulting in your service being attached to your selected VPC. These operations
+are not immediate, and also involve DNS changes which may take a few minutes to
+propagate.
+
+As mentioned on the confirmation modal, you will need to update your connection string
+in order to connect to your service after migration. The `Service URL` back on the
+service details page is already updated to include the new DNS info, and should be used
+for connecting to your service.
+
+<highlight type="tip">
+When migrating your service into a VPC, ensure that your AWS VPC's security groups
+allow network access from your AWS VPC to the Forge VPC which your service has been
+migrated into. Security group configuration was previously covered as part of
+peering connection setup. Double-check to be sure, otherwise you will not be able
+to connect to your Timescale Forge service.
+</highlight>
+
+### Migrate between VPCs
+When your service is already attached to a VPC, you have the option to migrate that
+service to another VPC within the same project.
+
+<img class="main-content__illustration" src="migrate-between-vpcs.png" alt="Migrate between VPCs"/>
+
+To migrate between VPCs, expand the `Migrate into another VPC` menu and select the VPC
+to migrate your service to. Then click `Migrate`. You will then be prompted to confirm
+the migration.
+
+<img class="main-content__illustration" src="migrate-between-vpcs-confirm.png" alt="Migrate between VPCs confirmation"/>
+
+Your service will now go through a series of asynchronous operations ultimately
+resulting in your service being detached from its previous VPC and attached to the
+new VPC which you selected.
+
+In the case of VPC to VPC migration, the `Service URL` connection string will not
+be updated, only the IP address which the DNS name is associated with will be updated.
```

Comment:
"In the case of VPC to VPC migration, only the IP address which the DNS name is associated with with will be updated."

Your next sentence makes it clear the connection string remains unchanged